diff --git a/src/Makefile.std b/src/Makefile.std
index 64fae25..330d11a 100644
--- a/src/Makefile.std
+++ b/src/Makefile.std
@@ -112,8 +112,8 @@ CC = gcc
 ##
 ## Standard -- "main-x11.c" & "main-gcu.c"
 ##
-CFLAGS = -Wall -O1 -pipe -g -D"USE_X11" -D"USE_GCU"
-LIBS = -lX11 -lcurses
+#CFLAGS = -Wall -O1 -pipe -g -D"USE_X11" -D"USE_GCU"
+#LIBS = -lX11 -lcurses
 
 
 ##
@@ -182,8 +182,8 @@ LIBS = -lX11 -lcurses
 ##
 ## Allows the use of Sil in the console using curses.
 ##
-#CFLAGS = -Wall -O2 -fno-strength-reduce -pipe -D"USE_GCU"
-#LIBS = -lcurses
+CFLAGS = -Wall -O2 -fno-strength-reduce -pipe -D"USE_GCU"
+LIBS = -lcurses
 
 
 ##
