diff --git a/include/config.h b/include/config.h
index 6ae1a7514..374163a97 100644
--- a/include/config.h
+++ b/include/config.h
@@ -55,7 +55,7 @@
 #if !defined(NOTTYGRAPHICS)
 #define TTY_GRAPHICS /* good old tty-based graphics */
 #endif
-/* #define CURSES_GRAPHICS *//* Curses interface - Karl Garrison*/
+#define CURSES_GRAPHICS /* Curses interface - Karl Garrison*/
 /* #define X11_GRAPHICS */   /* X11 interface */
 /* #define QT_GRAPHICS */    /* Qt interface */
 /* #define MSWIN_GRAPHICS */ /* Windows NT, CE, Graphics */
diff --git a/sys/unix/hints/linux.370 b/sys/unix/hints/linux.370
index e3dec785b..4d6677b13 100755
--- a/sys/unix/hints/linux.370
+++ b/sys/unix/hints/linux.370
@@ -341,7 +341,7 @@ CFLAGS+=-DSYSCF -DSYSCF_FILE=\"$(HACKDIR)/sysconf\" -DSECURE
 MOREALL=$(MAKE) install
 else #!WANT_SOURCE_INSTALL
 #PREFIX=/usr
-PREFIX=$(wildcard ~)/nh/install
+PREFIX=$(wildcard ~)/.local
 HACKDIR=$(PREFIX)/games/lib/nethackdir
 SHELLDIR = $(PREFIX)/games
 VARDIRPERM = 0755
